amundsen_application/static/js/components/DashboardPage/constants.ts

```diff
@@ -6,5 +6,8 @@ export const OWNER_HEADER_TEXT = 'Owners';
 export const ADD_DESC_TEXT = 'Add Description in';
 export const EDIT_DESC_TEXT = 'Click to edit description in';
 
+export const MISSED_RUN_TEXT = 'Missed';
+export const HIT_RUN_TEXT = 'Hit';
+
 export const LAST_RUN_SUCCEEDED = 'succeeded';
-export const LAST_RUN_FAILED = 'failed';
+
```

Comment:
Are your pre-commit hooks working at the moment? They should have fixed this automatically.

If they don't work, try to do `rm -rf node_modules && npm i`